custom/icds_reports/reports/awc_reports.py

```diff
@@ -1155,6 +1156,7 @@ def get_pregnant_details(case_id, awc_id):
         case_id=case_id,
         awc_id=awc_id,
         month__gte=ten_months_ago,
+        home_visit_date__lte=F('month') + timedelta(days=31),
```

Comment:
going to merge this so QA can continue but I think this can just be `home_vist_date__isnull=False` as home visit date should only be updated if there was a home visit in month